lib/tasks/represent_downstream.rake

```diff
@@ -55,13 +55,14 @@ namespace :represent_downstream do
   end
 
   desc "
-  Represent an individual edition downstream
+  Represent an individual or multiple editions downstream
   Usage
   rake 'represent_downstream:content_id[57a1253c-68d3-4a93-bb47-b67b9b4f6b9a]'
   "
   task :content_id, [:content_id] => :environment do |_t, args|
+    content_ids = args[:content_id].split(" ")
```

Comment:
It seems more intuitive to me that the separator would be a comma (if I did't read the docs/code, I would assume it to be a comma).